crates/aptos/e2e/main.py

```diff
@@ -117,6 +117,7 @@ def run_tests(run_helper):
 
     # Run move subcommand group tests.
     test_move_publish(run_helper)
+    test_move_compile(run_helper)
```

Comment:
Nit: Put this before publish, compile is just a subset of publish, it'd be nice to see first that compile succeeds if publish fails. It doesn't actually matter, it's just a brain thing hahah.